configure.ac

```diff
@@ -672,13 +672,13 @@ AC_TRY_COMPILE([],[
 )
 
 dnl if no environment variable is set, set the default value for the default intf
-if test "${xenable_intf_open}" = "xyes"; then
```

Comment:
I knew there was something wrong with this "xyes"... Good things don't look like [that](https://translate.google.com/#view=home&op=translate&sl=ru&tl=en&text=%D1%85%D1%83%D0%B5%D0%B2). ;) Still I missed it. Mea culpa. Good to have that fixed now.